Formula/yle-dl.rb

```diff
@@ -95,7 +95,7 @@ def install
   end
 
   test do
-    assert_equal "Traileri: 3 minuuttia-2012-05-30T10:51\n",
-                 shell_output("#{bin}/yle-dl --showtitle https://areena.yle.fi/1-1570236")
+    output = shell_output("#{bin}/yle-dl --showtitle https://areena.yle.fi/1-1570236")
+    assert_equal "Traileri: 1 minuutti-2012-05-30T10:51\n", output
```

Comment:
Can we replace this with a regex (maybe one that accepts any digit where appropriate) to make it less likely to break again?

Reply:
There's also a change in plural/singular form of minuuttia/minuutti 😄 
Because it's a video title (and it's now autogenerated I guess) It can be changed in any way. So I'd rather keep it as is or change video itself (which won't be protected for such kind of changes as well)